src/inference/src/dev/plugin.hpp

```diff
@@ -34,6 +34,8 @@ class Plugin {
 
     void set_name(const std::string& deviceName);
 
+    const std::string get_name() const;
```

Comment:
I propose to align names.
```suggestion
    const std::string& get_device_name() const;
```

Reply:
It is mirror api with:
`void set_name(const std::string& deviceName);`